tox.ini

```diff
@@ -5,53 +5,53 @@ skipsdist = true
 [testenv]
 deps = -r requirements-dev.txt
 commands_pre =
-    docker: docker exec es elasticsearch-plugin -s install analysis-icu
-    docker: docker restart es
+    docker: docker exec es-core elasticsearch-plugin -s install analysis-icu
+    docker: docker restart es-core
     python -m textblob.download_corpora
 commands =
-    pytest {posargs:--disable-warnings -v tests}
```

Comment:
Remove the `-v` flag from the test output, since we don't need verbose output, it just makes the test logs in github actions hard to read.